cordova-react/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "react-reddit-reader",
+  "version": "1.0.0",
+  "description": "A small web app in React that queries Reddit for submissions.",
+  "main": "index.js",
+  "dependencies": {
+    "jquery": "^2.2.1",
+    "react": "^0.14.7",
+    "react-dom": "^0.14.7"
```

Comment:
If you're already including these as dependencies, can you not avoid bundling files like `react-dom.min.js`? They should already be in your `node_modules` folder